### ISSUE TITLE
NVDA does not read cell contents in calc

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4564]
**Reported by aerospace on 2014-10-21 12:45**
when opening a spreadsheet in libreoffice calc (4.3.2) I can't get NVDA to announce either the cell coordinates or the contents in the cells.  As I arrow around the spreadsheet,  NVDA says "Input_Text Frame Editable Multi line."

I tried F2 to get to the input-box then escape to return to the cell-grid.  But the same behavior continued.

I experimented with arrowing around and copy/paste between calc and notepad to verify that the cursor does in fact move from cell to cell.  NVDA did move up, down, left and right as expected, but it only announced "Input_Text Frame Editable Multi line" as the cursor traveled from one cell to another.

I set NVDA's logging level to debug, opened an existing spreadsheet and arrowed around to capture a log durring the event.  Both the log and the spreadsheet used should be found attached to this ticket.

**Comment by surfer0627** [comment]
Come back to this issue.
Sometimes, while arrowing through cells:
braille: shows "cell"
speech: reads nothing
User needs to press alt twice to read cell contents.

This case just could be reproduced randamly. I open a ".ods" file and arrow through cells for a while until braille shows "cell" and speech reads nothing.
Is this also a LibreOffice bug?

Notes:
• LibreOffice 5.1.0.3

**Comment by surfer0627** [comment]
Please ignore previous comment. This case has been fixed in 5.1.2.2.

**Comment by surfer0627** [comment]
This bug still could be reproduced.
Here is some develop info from LibreOffice, please see [API for Supporting Assistive Technologies](http://www.libreoffice.org/get-help/accessibility/).

**Comment by zahra21** [comment]
hello.
the problem of supporting calc for screen reader users, is resolved completely in libreoffice 4.3.4.1
in some versions (4.3, 5.0) you need to use f2 and then scape first to make it accessible and navigate in cells.
in some versions like 4.4 and 5.1 you dont need even f2 for accessibility and reading content of the cells.
i think that this bug should be closed and fixed.
in 5.2, the first times which i installed it, i had many issues using calc, but this issue seems to resolved for me and i can use any version of calc since libreoffice version 4.3.4
hope that help, God bless you all.

**Comment by ehollig** [comment]
Thank you @zahra21 for testing these libreoffice issues. Closing as worksforme
